src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,115 @@
+| `SELECT ... FOR UPDATE / FOR SHARE` | ✓                                                   | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT`                            | —                                                   | ✓                                                | —                                                 | —                                                     | —                                             |
+| `INSERT ... RETURNING`              | ✓ ‡                                                 | ✓                                                | —                                                 | —                                                     | —                                             |
+| `UPDATE`                            | ✓ ‡                                                 | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `DELETE`                            | ✓ ‡                                                 | —                                                | —                                                 | —                                                     | ✓                                             |
+| `INSERT ... ON CONFLICT DO UPDATE`  | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `UPSERT`                            | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+
+‡ The `SELECT` check is only evaluated when the statement actually needs to read from the relation, e.g., in a `WHERE`, `SET`, or `RETURNING` clause that references table columns.
+
+Additional considerations include:
+
+- `SELECT` evaluation: CockroachDB always evaluates `SELECT` (`USING`) policies for `INSERT`, `UPDATE`, and `DELETE`, even when the statement doesn't reference table columns.
+- `ON CONFLICT ... DO NOTHING`: CockroachDB still runs constraint and row-level policy checks on the `VALUES` clause even when the candidate row is discarded because of a conflict. This is a known limitation described in [cockroachdb/cockroach#35370](https://github.com/cockroachdb/cockroach/issues/35370).
```

Comment:
We do the opposite: we _don't_ evaluate the policy check of the VALUES clause if there is a conflict. Unlike postgres, which does evaluate them.

Reply:
ah ok i've updated to the following, please let me know if this needs more work 

> - `ON CONFLICT ... DO NOTHING`: CockroachDB does not run the constraint and row-level policy checks on the `VALUES` clause if the candidate row has a conflict. This is a known limitation described in [cockroachdb/cockroach#35370](https://github.com/cockroachdb/cockroach/issues/35370).